home/src/app/theme/components/searchbar/searchbarCtrl.js

```diff
@@ -28,6 +28,14 @@
 				$rootScope.mapPage = true;
 				$rootScope.mapSearchStringTrans = $scope.searchString;
 				$state.go('map', {}, { reload: true });
+			} else {
+				$location.path("/search");
+				if ($location.path() == "/search") {
+					searchService.prepForBroadcast($rootScope.predicates);
+					$scope.$on('handleBroadcast', function() {
+				        searchService.setBroadcast($rootScope.predicates);
```

Comment:
I already change value of $rootScope.predicates each time user type anything in search bar, so, you don't have to do this again